relayer/chain/ethereum/transact_opts_test.go

```diff
@@ -0,0 +1,248 @@
+package ethereum_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/snowfork/snowbridge/relayer/chain/ethereum"
+	"github.com/snowfork/snowbridge/relayer/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCalculateFee_NoFeesSpecified(t *testing.T) {
+	config := config.EthereumConfig{
+		GasFeeCap:        0,
+		GasTipCap:        0,
+		GasFeeMultiplier: 0,
+		GasTipMultiplier: 0,
+	}
+
+	var suggestedFee *big.Int
+	var suggestedTip *big.Int
+
+	fee := ethereum.CalculateFee(config, suggestedFee, suggestedTip)
+
+	assert.Nil(t, fee.GasFeeCap)
+	assert.Nil(t, fee.GasTipCap)
```

Comment:
I guess you can convert these to table driven tests, but it is not really necessary.

Reply:
Is this like a library? Or do I just create a table using an array and compress it all into one test?